cluster/gce/config-common.sh

```diff
@@ -124,7 +124,7 @@ function get-alias-range-size() {
 }
 # NOTE: Avoid giving nodes empty scopes, because kubelet needs a service account
 # in order to initialize properly.
-NODE_SCOPES="${NODE_SCOPES:-monitoring,logging-write,storage-ro}"
+NODE_SCOPES="${NODE_SCOPES:-monitoring,logging-write,storage-ro,trace}"
```

Comment:
does this take effect for test and non-test clusters? I would expect config changes to escape test configuration for this feature yet

Reply:
I can remove these changes.

Reply:
They were just for convenience, but aren't required for the e2e test.